Error: Cannot find module '@azure/communication-configuration'

<!--
IF SUFFICIENT INFORMATION IS NOT PROVIDED VIA THE FOLLOWING TEMPLATE THE ISSUE MIGHT BE CLOSED WITHOUT FURTHER CONSIDERATION OR INVESTIGATION
-->
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> npm install
npm run start

### Any log messages given by the failure
>
hyounoo@Hyounwoos-MacBook-Pro contoso-node-api % npm run start                           

> acs-node-project@1.0.0 start /Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api
> node ./bin/www

connecting to cosmosdb...
internal/modules/cjs/loader.js:834
  throw err;
  ^

Error: Cannot find module '@azure/communication-configuration'
Require stack:
- /Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/controllers/appointments.controller.js
- /Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/routes/appointments.routes.js
- /Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/app.js
- /Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/bin/www
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/controllers/appointments.controller.js:1:25)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
    at Module.load (internal/modules/cjs/loader.js:879:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:903:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/controllers/appointments.controller.js',
    '/Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/routes/appointments.routes.js',
    '/Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/app.js',
    '/Users/hyounoo/Github/communication-services-contoso-med-app/contoso-node-api/bin/www'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! acs-node-project@1.0.0 start: `node ./bin/www`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the acs-node-project@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/hyounoo/.npm/_logs/2020-12-10T03_15_26_115Z-debug.log
hyounoo@Hyounwoos-MacBook-Pro contoso-node-api % 


### Expected/desired behavior
> Expected to run the api server

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)
macOS Catalina
### Versions
> 10.15.7

### Mention any other details that might be useful
Had some issue configuration config.json without detailed documentation.
I could create Azure CosmosDB with MongoDB Api option but the document didn't specified which to choose.

AlsoI couldn't fill the  jwtPrivateKey as there is no explanation where to get this token key.

It would be better if the document describes how to deploy this to azure with step by step description(with some pictures :D).
> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

--- a/contoso-node-api/controllers/appointments.controller.js
+++ b/contoso-node-api/controllers/appointments.controller.js
@@ -1,7 +1,5 @@
-const UserTokenClient = require("@azure/communication-configuration").UserTokenClient;
 const appointmentsService = require('../services/appointments.service');
 const config = require("../config.json");
-const configurationClient = new UserTokenClient(config.connectionString);
 
 if (!config || !config.connectionString || config.connectionString.indexOf('endpoint=') === -1) {
     throw new Error("Update `config.json` with connection string");

--- a/contoso-node-api/db/dbInitialize.js
+++ b/contoso-node-api/db/dbInitialize.js
@@ -27,13 +27,19 @@ const initializeDB = async () => {
         doctor.slots.push(...generateSlots())
     })
 
-    console.log('adding new doctors collection')
-    await db.collection("Doctors").insertMany(doctorUsers)
-    console.log('added doctors users')
+    try {
+        console.log('adding new doctors collection')
+        await db.collection("Doctors").insertMany(doctorUsers)
+        console.log('added doctors users')
 
-    console.log('adding new patients collection')
-    await db.collection("Patients").insertMany(patientUsers)
-    console.log('added patient users')
+        console.log('adding new patients collection')
+        await db.collection("Patients").insertMany(patientUsers)
+        console.log('added patient users')
+
+        console.log('Database initialization completed successfully.')
+    } catch (e) {
+        console.log(e)
+    }
 }
 
 const SLOTS_PER_DAY = 5;

--- a/contoso-node-api/package.json
+++ b/contoso-node-api/package.json
@@ -1,6 +1,6 @@
 {
   "name": "acs-node-project",
-  "version": "1.0.0",  
+  "version": "1.0.0",
   "author": "Microsoft Corporation",
   "license": "MIT",
   "scripts": {

